Source/Core/Core/Core.cpp

```diff
@@ -334,6 +354,9 @@ void EmuThread()
 	DisplayMessage(cpu_info.Summarize(), 8000);
 	DisplayMessage(core_parameter.m_strFilename, 3000);
 
+	//For a time this acts as the CPU thread...
+	DeclareAsCPUThread();
```

Comment:
Yeah, I did check, but this point in code was the latest I could DeclareAsCPUThread(). Apparently something was calling the SetInterrupt() before the CBoot::BootUp(), and I'm not exactly sure what. You could probably call the UndeclareAsCPUThread() inside of CBoot::BootUp(), but I figured I should keep the code relatively close so it's easier to follow.